### DISCUSSION TITLE
Can No Longer Assign Multiple Users to Issue

### DISCUSSION THREAD

**Discussion by macbraughton** [community/community#78991] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I noticed last night that you can no longer assign multiple users to issues. I was just reading about the MySQL internal upgrade that just happened and it definitely smells to me like a bug that could be related to the migration? This is merely inconvenient for me but could be a huge issue for people working with larger teams.

This feature worked for me as recently as a few weeks ago, so this is a new bug.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by erickdc7** [comment]
It seems like you've encountered a new bug where you can't assign multiple users to an issue, which was possible until a few weeks ago. This might be related to the recent MySQL internal upgrade. It's definitely worth reporting as it affects not only your workflow but could also impact larger teams significantly.

I suggest reaching out to the support team or submitting a bug report detailing your observations. It's important to mention that this feature was working recently and to inquire if it's an unintended side effect of the upgrade.

**Reply by macbraughton** [reply]
Thanks, yes, I would love to report it but am having trouble figuring out where to do it, that's why I posted it here, can you help point me in the right direction?
